interop-testing/src/main/java/io/grpc/testing/integration/TestServiceClient.java

```diff
@@ -239,6 +256,15 @@ void parseArgs(String[] args) throws Exception {
           + "\n                              should stop and fail, if the desired number of "
           + "\n                              iterations have not yet completed. Default "
             + c.soakOverallTimeoutSeconds
+          + "\n --enable_observability=true|false "
+          + "                                Whether to enable GCP Observability. Default "
+            + TestServiceClient.enableObservability
+          // TODO(stanleycheung): remove this param once all the observability exporter plugins
```

Comment:
Don't have this at all. This should be controlled by this implementation, not the test framework. This code has much better knowledge of the upload frequency than the test framework. Just hard-code the appropriate sleep into the code when observability is enabled.

Reply:
Implemented. Also remove that from the framework and other lang's implementation. Right now it's hardcoded at 65 seconds everywhere.